src/azul/service/manifest_service.py

```diff
@@ -622,8 +639,37 @@ def __init_subclass__(cls) -> None:
             cls.cls_for_format[format] = cls
 
     @classmethod
-    def command_lines(cls, url: str) -> Optional[JSON]:
-        return None
+    def _cmd_exe_quote(cls, s: str) -> str:
+        """
+        Escape a string for insertion into a `cmd.exe` command line
+        """
+        assert '"' not in s, s
+        assert '\\' not in s, s
+        return f'"{s}"'
+
+    @classmethod
+    def command_lines(cls, url: str, file_name: str) -> Optional[JSON]:
+        # A URL longer than 255 chars triggers a bug in curl.exe's
+        # implementation of '-O' on Windows. This is especially surprising
+        # because '-O' doesn't need to parse the URL when '-J' is also passed.
```

Comment:
inconsistent with code below

Reply:
Use long flag format instead of short. 
Prefix comment, 'Normally we would have used…, but we opted for…'